realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -593,48 +569,6 @@ private boolean isPrimaryKeyColumn(long columnIndex) {
     }
 
     /**
-     * Returns a view sorted by the specified column and order.
-     *
-     * @param columnIndex the column index.
-     * @param sortOrder the sort order.
-     * @return a sorted view.
-     */
-    public TableView getSortedView(long columnIndex, Sort sortOrder){
```

Comment:
@Zhuinden , nope, the `where()` created a `Query` object which is building `RealmResults` from `Query` not the `Table`.